tests/Integration/Behaviour/Features/Scenario/Product/update_stock.feature

```diff
@@ -254,3 +254,33 @@ Feature: Update product stock from Back Office (BO)
     And product "product1" localized "available_later_labels" should be:
       | locale | value |
       | en-US  |       |
+
+  Scenario: I update product to max and min stock
+    Given I add product "product2" with following information:
+      | name[en-US] | product2 |
+      | type        | standard |
+    And product "product2" type should be standard
+    And product "product2" should have following stock information:
+      | quantity | 0 |
+    When I update product "product2" stock with following information:
+      | delta_quantity | -2147483648 |
+    And product "product2" should have following stock information:
+      | quantity | -2147483648 |
+    When I update product "product2" stock with following information:
+      | delta_quantity | 4294967295 |
+    And product "product2" should have following stock information:
+      | quantity | 2147483647 |
```

Comment:
You should add changes with the biggest range -4294967295 and 4294967295 and heck the stock movements values like in update_combination_stock.feature

Reply:
Sorry my comment wasn't clear enough, w need to check that we can use the maximum delta values here, you already tested with `delta_quantity = 4294967295` you need to add one with `delta_quantity = -4294967295`
And at then end you need to use the stock movement step like:

```
    And product "product2" last stock movements should be:
      | employee   | delta_quantity |
      | Puff Daddy | -4294967295 |
      | Puff Daddy | 4294967295 |
      | Puff Daddy | -2147483648 |
```

Reply:
Sorry to be picky about this one, it's to make sure that:
- we can actually use the maximum range
- that it is correctly saved in the stock movement table
- that the DB won't be modified and change our max range without notice